src/planning/strategies/match-recipe-by-verb.ts

```diff
@@ -133,15 +143,17 @@ export class MatchRecipeByVerb extends Strategy {
                 if (name) {
                   if (tryApplyHandleConstraint(
```

Comment:
style nit: I don't like these spread-out function calls, would prefer we re-collapse them when we encounter them.

Reply:
agreed. for now i just removed the one parameter that could be removed without significant method refactoring.